m4/src/evaluate/mod.rs

```diff
@@ -0,0 +1,2085 @@
+use std::cell::RefCell;
```

Comment:
The normal Rust preference these days is to avoid putting notable program logic in a `mod.rs`, as it can sometimes be overlooked by humans assuming a small mod.rs.

Reply:
@jgarzik updated, let me know if you think this is an improvement, I've re-organised it quite a bit